### DISCUSSION TITLE
Removal of {{Pref}} macro

### DISCUSSION THREAD

**Discussion by teoli2003** [mdn/mdn-community#72] [Localization] [answered]
Hi all!

The `{{Pref}}` macro has being [removed from mdn/content](https://github.com/mdn/content/pull/https://github.com/mdn/content/pull/15761) and [deprecated in mdn/yari](https://github.com/mdn/yari/pull/6241). 

I also opened an issue for translators: mdn/translated-content#5565 to track the work in the translated content.

Once this is done, we will be able to delete this macro from Yari. 🎉 

Thanks all for your help.

**Comment by yin1999** [comment]
Created mdn/translated-content#5567 for `l10n-zh`.

**Comment by teoli2003** [comment, marked as answer]
Thanks all the occurrences of this macro are gone!
